zwave_js_server/util/command_class/energy_production.py

```diff
@@ -0,0 +1,39 @@
+"""Energy Production Command Class specific utility functions for values."""
+from __future__ import annotations
+
+from ...const import CommandClass
+from ...const.command_class.energy_production import (
+    CC_SPECIFIC_PARAMETER,
+    CC_SPECIFIC_SCALE,
+    ENERGY_PRODUCTION_PARAMETER_TO_SCALE_ENUM_MAP,
+    EnergyProductionParameter,
+    EnergyProductionScaleType,
+)
+from ...exceptions import InvalidCommandClass, UnknownValueData
+from ...model.value import Value
+
+
+def get_energy_production_parameter(value: Value) -> EnergyProductionParameter:
+    """Get the EnergyProductionParameter for a given value."""
+    if value.command_class != CommandClass.ENERGY_PRODUCTION:
+        raise InvalidCommandClass(value, CommandClass.ENERGY_PRODUCTION)
+    try:
+        return EnergyProductionParameter(
+            value.metadata.cc_specific[CC_SPECIFIC_PARAMETER]
+        )
+    except ValueError:
+        raise UnknownValueData(  # pylint: disable=raise-missing-from
```

Comment:
I think you can use `raise from None` if you want to disable the exception chaining.